docs/features/shared-volumes-with-NFS.md

```diff
@@ -0,0 +1,129 @@
+# Supporting shared volumes with vSphere Docker Volumes
+
+## Requirements
+* Developer should able to use externally managed NFS share as “shared docker volume” using “vsphere” volume driver.
+* This allows them to use NFS directories as docker volumes using Docker APIs, Compose etc 
+* Extend the vSphere Docker Volume plugin to support NFS shared volumes and mutliple readers/writers access
+* Volumes are accessible on multiple docker hosts running on  multiple ESX hosts in a Docker swarm
+* Allow admin to specify what remote dirs are managed by the plugin
+* Support all current docker volume plugin API for shared volumes
+
+## User experience
+### Set up remote directories for docker volumes
```

Comment:
This part is not very clear to me:

1. Exported remote directories...
Are we talking about setting up the remote NFS server here?

2. Configured sufficient permissions for the admins...
How do we configure NFS access permissions? Using docker-volume-vsphere.conf?

3. "The docker host has the necessary packages of the desired version of NFS installed."
Should be "...NFS client installed"?

4. "The admin can use ..."
This item seems inconsistent with above 3 items. Above 3 items are talking about something admin SHOULD do before creating shared volumes. But this item is talking about something admin CAN do with shared volumes feature.